tools/vscode_settings.py

```diff
@@ -13,7 +14,7 @@
     HAS_JSON5 = False
 
 
-ROOT_FOLDER = Path(__file__).absolute().parent.parent
+ROOT_FOLDER = Path(__file__).resolve().parent.parent
```

Comment:
Absolute and resolve are not the same, are they? And I think `absolute` is what is needed here, isn't it?
```suggestion
ROOT_FOLDER = Path(__file__).resolve().parents[1]
```